### ISSUE TITLE
Dimenstion mismatch error

### ISSUE THREAD

**Issue by Shoshin23** [ShafeenTejani/fast-style-transfer#2]
Hey Shafeen, 

I'm trying to run the code out of the box but i'm unable to get it to work. I'm attaching a pic of the stacktrace. Please let me know how i can fix this. I'll submit a PR if I can trace the issue.

![1__bash_and_new_issue_ _shafeentejani_fast-style-transfer](https://cloud.githubusercontent.com/assets/2094133/25478790/17ac9154-2b60-11e7-889f-3c1feb263c0a.jpg)

**Comment by thiagolcks** [comment]
Hi! I'm having the same issue here. When I change the image size it usually works.

**Comment by ShafeenTejani** [comment]
Hi shoshin and thiago,

Thanks for finding this bug. The error you were getting was caused by the fact that the input image dimensions were not divisible by 4 which causes an issue when trying to apply two consecutive levels of transpose convolution both with a stride of 2. I've pushed a fix to trim the input image to have the right dimensions so it should work now!

https://github.com/ShafeenTejani/fast-style-transfer/commit/2915fc4fe6269770ae748ccc1fefaef46e171921
